src/components/Sign/Signup.js

```diff
@@ -90,233 +94,266 @@ class SignUp extends Component {
+                      </Form.Group>
+                      <Form.Group
+                        as={Col}
+                        md="12"
+                        controlId="validationPassword"
+                      >
+                        <Form.Label className="text-muted">Password</Form.Label>
+                        <InputGroup>
+                          <Form.Control
+                            type={this.state.showPassword ? "text" : "password"}
+                            name="password"
+                            placeholder="password"
+                            value={values.password}
+                            onChange={handleChange}
+                            isValid={touched.password && !errors.password}
+                            isInvalid={!!errors.password}
+                          />
+                          <InputGroup.Prepend>
+                            <div
+                              className="col-1 text-center icon"
+                              onClick={this.toggleShow}
+                            >
+                              <FaEye id="signup-icon-eye" size={32} />
+                            </div>
+                          </InputGroup.Prepend>
+                          <Form.Control.Feedback
+                            as={Col}
+                            md="12"
+                            type="invalid"
+                          >
+                            {errors.password}
+                          </Form.Control.Feedback>
+                        </InputGroup>
+                      </Form.Group>
+                      <Form.Group
+                        as={Col}
+                        md="12"
+                        controlId="validationConfirmPassword"
+                      >
+                        <Form.Label className="text-muted">
+                          Confirm Password
+                        </Form.Label>
+                        <InputGroup>
+                          <Form.Control
+                            type={
+                              this.state.showPasswordConfirm
+                                ? "text"
+                                : "password"
+                            }
+                            name="passwordConfirm"
+                            placeholder="confirm password"
+                            value={values.passwordConfirm}
+                            onChange={handleChange}
+                            isValid={
+                              touched.passwordConfirm && !errors.passwordConfirm
+                            }
+                            isInvalid={!!errors.passwordConfirm}
+                          />
+                          <InputGroup.Prepend>
+                            <div
+                              className="col-1 text-center icon"
+                              onClick={this.toggleShowConfirm}
+                            >
+                              <FaEye size={32} />
```

Comment:
Aquí le había puesto un id para las pruebas que está haciendo David

Reply:
![image](https://user-images.githubusercontent.com/39096718/66229083-a9d85f00-e6a6-11e9-99d3-3d5a8c1e3580.png)

Reply:
Hay dos <FaEye> en los dos debería estar ese id?
La diferencia es que uno esta despues de este codigo`<div
                              className="col-1 text-center icon"
                              onClick={this.toggleShowConfirm}
                            >`
Y el otro despues de este `<div
                              className="col-1 text-center icon"
                              onClick={this.toggleShow}
                            >` 
En ambos casos (lo que se borro y lo que se tiene), el unico que tiene ID es el que va despues de `<div
                              className="col-1 text-center icon"
                              onClick={this.toggleShow}
                            >`